logger/text_formatter.go

```diff
@@ -0,0 +1,65 @@
+// pmm-admin
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package logger provides helpers for logger.
+package logger
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// TextFormatter formats logs into text.
+type TextFormatter struct {
+	*logrus.TextFormatter
+}
+
+// Format renders a single log entry.
+func (f *TextFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	if entry.Level == logrus.FatalLevel {
```

Comment:
We need other levels too.
If the only way is to copy/paste logrus TextFormatter – so be it, just make sure to add a comment about this. And please check that when --trace is passed and SetReportCaller is enabled, the caller is reported correctly.

Reply:
It's not possible to copy only TextFormatter, because it calls internal functions and fields. I found other way to remove log level prefix.